index.d.ts

```diff
@@ -1584,6 +1584,29 @@ declare namespace Moleculer {
 			delimiter(value: string): void;
 		}
 	}
+
+	namespace Utils {
+		function isFunction(func: any): boolean;
```

Comment:
The "is" series of functions are basically acting as typeguards, and we can express that in the syntax here.  For instance, `isString(str: unknown): str is string;`

I also think its best to use `unknown` instead of `any` on these as it more appropriately describes the expected inputs.

Reply:
I'm unfamiliar with this syntax. I have updated the PR. Please let me know if this is correct. Thank you!

Reply:
Sorry, I should have looked at your replies here before I did a follow up review.  You did great with the typeguard syntax.  If you're interested, here are some notes on it: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-1-6.html#user-defined-type-guard-functions.  (Side-note, the typeguard syntax also works for things like array filters to narrow the result).

The function one I noted in my follow-up review.

Reply:
Thank you for this reference. 👍